frontend/src/components/ShootDetails/ShootAccessCard.vue

```diff
@@ -303,7 +303,7 @@ export default {
     overflow-x: auto;
   }
 
-  .wrap-text {
+  .wrap-code {
```

Comment:
This change is not needed.

Reply:
yes, I know. But I wanted to highlight that this is not only wrap text but has more css styles. But I will change it back to .wrap-text, no problem.